tests/it-test.js

```diff
@@ -67,13 +67,13 @@ describe('it', function() {
   });
 
   it('skips tests with the .skip modifier', function() {
-    expect(skippedError).to.be.null;
+    expect(skippedError).to.be.null();
     var pendingSpec = Mocha.suite.suites.find(function(suite) {
       return suite.tests.find(function(test) {
-        return test.title === 'a skipped spec';
+        return test.title === 'is a skipped spec';
       });
     });
-    expect(pendingSpec).to.be.defined;
```

Comment:
This was actually never doing the intended verification of  `pendingSpec`. `defined` is actually not a defined chai assertion and so this was testing for `undefined`. The new function syntax prevents those errors.